```
2023-07-31T06:56:53.1867632Z --- PASS: TestEncodeDecodeTierConfigMgr (0.00s)
2023-07-31T06:56:53.1867777Z === RUN   TestPrepareUpdateMessage
2023-07-31T06:56:53.1868104Z     update-notifier_test.go:88: Testcase 1: output '
2023-07-31T06:56:53.1868551Z          You are running an older version of MinIO released 3 days before the latest release 
2023-07-31T06:56:53.1868891Z          Update: my_download_url 
2023-07-31T06:56:53.1869046Z         
2023-07-31T06:56:53.1869652Z         ' did not contain line 1: 'You are running an older version of MinIO released 3 days ago'
2023-07-31T06:56:53.1869962Z     update-notifier_test.go:88: Testcase 2: output '
2023-07-31T06:56:53.1870424Z          You are running an older version of MinIO released 3 hours before the latest release 
2023-07-31T06:56:53.1871157Z          Update: https://my_download_url_is_huge/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa 
2023-07-31T06:56:53.1871268Z         
2023-07-31T06:56:53.1872149Z         ' did not contain line 1: 'You are running an older version of MinIO released 3 hours ago'
2023-07-31T06:56:53.1872481Z     update-notifier_test.go:88: Testcase 7: output '
2023-07-31T06:56:53.1872933Z          You are running an older version of MinIO released 1 second before the latest release 
2023-07-31T06:56:53.1873125Z          Update: my_download_url 
2023-07-31T06:56:53.1873200Z         
2023-07-31T06:56:53.1873738Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 second ago'
2023-07-31T06:56:53.1874045Z     update-notifier_test.go:88: Testcase 8: output '
2023-07-31T06:56:53.1874714Z          You are running an older version of MinIO released 37 seconds before the latest release 
2023-07-31T06:56:53.1875097Z          Update: my_download_url 
2023-07-31T06:56:53.1875190Z         
2023-07-31T06:56:53.1876151Z         ' did not contain line 1: 'You are running an older version of MinIO released 37 seconds ago'
2023-07-31T06:56:53.1876560Z     update-notifier_test.go:88: Testcase 9: output '
2023-07-31T06:56:53.1876994Z          You are running an older version of MinIO released 1 minute before the latest release 
2023-07-31T06:56:53.1877188Z          Update: my_download_url 
2023-07-31T06:56:53.1877280Z         
2023-07-31T06:56:53.1878013Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 minute ago'
2023-07-31T06:56:53.1878433Z     update-notifier_test.go:88: Testcase 10: output '
2023-07-31T06:56:53.1878881Z          You are running an older version of MinIO released 1 minute before the latest release 
2023-07-31T06:56:53.1879075Z          Update: my_download_url 
2023-07-31T06:56:53.1879165Z         
2023-07-31T06:56:53.1879699Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 minute ago'
2023-07-31T06:56:53.1880009Z     update-notifier_test.go:88: Testcase 11: output '
2023-07-31T06:56:53.1880472Z          You are running an older version of MinIO released 37 minutes before the latest release 
2023-07-31T06:56:53.1880663Z          Update: my_download_url 
2023-07-31T06:56:53.1880777Z         
2023-07-31T06:56:53.1881645Z         ' did not contain line 1: 'You are running an older version of MinIO released 37 minutes ago'
2023-07-31T06:56:53.1882167Z     update-notifier_test.go:88: Testcase 12: output '
2023-07-31T06:56:53.1882597Z          You are running an older version of MinIO released 1 hour before the latest release 
2023-07-31T06:56:53.1882942Z          Update: my_download_url 
2023-07-31T06:56:53.1883075Z         
2023-07-31T06:56:53.1883658Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 hour ago'
2023-07-31T06:56:53.1883996Z     update-notifier_test.go:88: Testcase 13: output '
2023-07-31T06:56:53.1884437Z          You are running an older version of MinIO released 1 hour before the latest release 
2023-07-31T06:56:53.1884628Z          Update: my_download_url 
2023-07-31T06:56:53.1884714Z         
2023-07-31T06:56:53.1885539Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 hour ago'
2023-07-31T06:56:53.1885891Z     update-notifier_test.go:88: Testcase 14: output '
2023-07-31T06:56:53.1886339Z          You are running an older version of MinIO released 2 hours before the latest release 
2023-07-31T06:56:53.1886646Z          Update: my_download_url 
2023-07-31T06:56:53.1886741Z         
2023-07-31T06:56:53.1887271Z         ' did not contain line 1: 'You are running an older version of MinIO released 2 hours ago'
2023-07-31T06:56:53.1887578Z     update-notifier_test.go:88: Testcase 15: output '
2023-07-31T06:56:53.1888027Z          You are running an older version of MinIO released 1 day before the latest release 
2023-07-31T06:56:53.1888202Z          Update: my_download_url 
2023-07-31T06:56:53.1888293Z         
2023-07-31T06:56:53.1889038Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 day ago'
2023-07-31T06:56:53.1889617Z     update-notifier_test.go:88: Testcase 16: output '
2023-07-31T06:56:53.1890375Z          You are running an older version of MinIO released 1 day before the latest release 
2023-07-31T06:56:53.1890609Z          Update: my_download_url 
2023-07-31T06:56:53.1890698Z         
2023-07-31T06:56:53.1891268Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 day ago'
2023-07-31T06:56:53.1891596Z     update-notifier_test.go:88: Testcase 17: output '
2023-07-31T06:56:53.1892228Z          You are running an older version of MinIO released 2 days before the latest release 
2023-07-31T06:56:53.1892443Z          Update: my_download_url 
2023-07-31T06:56:53.1892683Z         
2023-07-31T06:56:53.1893282Z         ' did not contain line 1: 'You are running an older version of MinIO released 2 days ago'
2023-07-31T06:56:53.1893609Z     update-notifier_test.go:88: Testcase 18: output '
2023-07-31T06:56:53.1894055Z          You are running an older version of MinIO released 1 week before the latest release 
2023-07-31T06:56:53.1894231Z          Update: my_download_url 
2023-07-31T06:56:53.1894321Z         
2023-07-31T06:56:53.1894878Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 week ago'
2023-07-31T06:56:53.1895210Z     update-notifier_test.go:88: Testcase 19: output '
2023-07-31T06:56:53.1895994Z          You are running an older version of MinIO released 1 week before the latest release 
2023-07-31T06:56:53.1896257Z          Update: my_download_url 
2023-07-31T06:56:53.1896349Z         
2023-07-31T06:56:53.1896961Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 week ago'
2023-07-31T06:56:53.1897465Z     update-notifier_test.go:88: Testcase 20: output '
2023-07-31T06:56:53.1897927Z          You are running an older version of MinIO released 2 weeks before the latest release 
2023-07-31T06:56:53.1898116Z          Update: my_download_url 
2023-07-31T06:56:53.1898205Z         
2023-07-31T06:56:53.1898787Z         ' did not contain line 1: 'You are running an older version of MinIO released 2 weeks ago'
2023-07-31T06:56:53.1899282Z     update-notifier_test.go:88: Testcase 21: output '
2023-07-31T06:56:53.1899747Z          You are running an older version of MinIO released 1 month before the latest release 
2023-07-31T06:56:53.1899943Z          Update: my_download_url 
2023-07-31T06:56:53.1900019Z         
2023-07-31T06:56:53.1900611Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 month ago'
2023-07-31T06:56:53.1901051Z     update-notifier_test.go:88: Testcase 22: output '
2023-07-31T06:56:53.1901509Z          You are running an older version of MinIO released 1 month before the latest release 
2023-07-31T06:56:53.1901699Z          Update: my_download_url 
2023-07-31T06:56:53.1901786Z         
2023-07-31T06:56:53.1902309Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 month ago'
2023-07-31T06:56:53.1902813Z     update-notifier_test.go:88: Testcase 23: output '
2023-07-31T06:56:53.1903303Z          You are running an older version of MinIO released 2 months before the latest release 
2023-07-31T06:56:53.1903499Z          Update: my_download_url 
2023-07-31T06:56:53.1903588Z         
2023-07-31T06:56:53.1904547Z         ' did not contain line 1: 'You are running an older version of MinIO released 2 months ago'
2023-07-31T06:56:53.1904946Z     update-notifier_test.go:88: Testcase 24: output '
2023-07-31T06:56:53.1905387Z          You are running an older version of MinIO released 1 year before the latest release 
2023-07-31T06:56:53.1905579Z          Update: my_download_url 
2023-07-31T06:56:53.1905665Z         
2023-07-31T06:56:53.1906376Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 year ago'
2023-07-31T06:56:53.1906781Z     update-notifier_test.go:88: Testcase 25: output '
2023-07-31T06:56:53.1907990Z          You are running an older version of MinIO released 1 year before the latest release 
2023-07-31T06:56:53.1908193Z          Update: my_download_url 
2023-07-31T06:56:53.1908284Z         
2023-07-31T06:56:53.1908818Z         ' did not contain line 1: 'You are running an older version of MinIO released 1 year ago'
2023-07-31T06:56:53.1909143Z     update-notifier_test.go:88: Testcase 26: output '
2023-07-31T06:56:53.1909939Z          You are running an older version of MinIO released 2 years before the latest release 
2023-07-31T06:56:53.1910208Z          Update: my_download_url 
2023-07-31T06:56:53.1910300Z         
2023-07-31T06:56:53.1910909Z         ' did not contain line 1: 'You are running an older version of MinIO released 2 years ago'
2023-07-31T06:56:53.1911641Z --- FAIL: TestPrepareUpdateMessage (0.00s)
```

--- a/cmd/update-notifier_test.go
+++ b/cmd/update-notifier_test.go
@@ -35,36 +35,36 @@ func TestPrepareUpdateMessage(t *testing.T) {
 		expectedSubStr string
 	}{
 		// Testcase index 0
-		{72 * time.Hour, "my_download_url", "3 days ago"},
-		{3 * time.Hour, "https://my_download_url_is_huge/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "3 hours ago"},
+		{72 * time.Hour, "my_download_url", "3 days before the latest release"},
+		{3 * time.Hour, "https://my_download_url_is_huge/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "3 hours before the latest release"},
 		{-72 * time.Hour, "another_update_url", ""},
 		{0, "another_update_url", ""},
 		{time.Hour, "", ""},
 		{0 * time.Second, "my_download_url", "now"},
-		{1 * time.Second, "my_download_url", "1 second ago"},
-		{37 * time.Second, "my_download_url", "37 seconds ago"},
-		{60 * time.Second, "my_download_url", "1 minute ago"},
-		{61 * time.Second, "my_download_url", "1 minute ago"},
+		{1 * time.Second, "my_download_url", "1 second before the latest release"},
+		{37 * time.Second, "my_download_url", "37 seconds before the latest release"},
+		{60 * time.Second, "my_download_url", "1 minute before the latest release"},
+		{61 * time.Second, "my_download_url", "1 minute before the latest release"},
 
 		// Testcase index 10
-		{37 * time.Minute, "my_download_url", "37 minutes ago"},
-		{1 * time.Hour, "my_download_url", "1 hour ago"},
-		{61 * time.Minute, "my_download_url", "1 hour ago"},
-		{122 * time.Minute, "my_download_url", "2 hours ago"},
-		{24 * time.Hour, "my_download_url", "1 day ago"},
-		{25 * time.Hour, "my_download_url", "1 day ago"},
-		{49 * time.Hour, "my_download_url", "2 days ago"},
-		{7 * 24 * time.Hour, "my_download_url", "1 week ago"},
-		{8 * 24 * time.Hour, "my_download_url", "1 week ago"},
-		{15 * 24 * time.Hour, "my_download_url", "2 weeks ago"},
+		{37 * time.Minute, "my_download_url", "37 minutes before the latest release"},
+		{1 * time.Hour, "my_download_url", "1 hour before the latest release"},
+		{61 * time.Minute, "my_download_url", "1 hour before the latest release"},
+		{122 * time.Minute, "my_download_url", "2 hours before the latest release"},
+		{24 * time.Hour, "my_download_url", "1 day before the latest release"},
+		{25 * time.Hour, "my_download_url", "1 day before the latest release"},
+		{49 * time.Hour, "my_download_url", "2 days before the latest release"},
+		{7 * 24 * time.Hour, "my_download_url", "1 week before the latest release"},
+		{8 * 24 * time.Hour, "my_download_url", "1 week before the latest release"},
+		{15 * 24 * time.Hour, "my_download_url", "2 weeks before the latest release"},
 
 		// Testcase index 20
-		{30 * 24 * time.Hour, "my_download_url", "1 month ago"},
-		{31 * 24 * time.Hour, "my_download_url", "1 month ago"},
-		{61 * 24 * time.Hour, "my_download_url", "2 months ago"},
-		{360 * 24 * time.Hour, "my_download_url", "1 year ago"},
-		{361 * 24 * time.Hour, "my_download_url", "1 year ago"},
-		{2 * 365 * 24 * time.Hour, "my_download_url", "2 years ago"},
+		{30 * 24 * time.Hour, "my_download_url", "1 month before the latest release"},
+		{31 * 24 * time.Hour, "my_download_url", "1 month before the latest release"},
+		{61 * 24 * time.Hour, "my_download_url", "2 months before the latest release"},
+		{360 * 24 * time.Hour, "my_download_url", "1 year before the latest release"},
+		{361 * 24 * time.Hour, "my_download_url", "1 year before the latest release"},
+		{2 * 365 * 24 * time.Hour, "my_download_url", "2 years before the latest release"},
 	}
 
 	plainMsg := "You are running an older version of MinIO released"

